### PR TITLE
pr-portfolio-works-popup-with-js

### PR DESCRIPTION
**Pull request by ivanmvh**
Implement the following interactions:

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
Refactor the project section:
-- use a JavaScript array to store all of the information for all projects.
-- For each project we store the following pieces of data in a JavaScript object, at a minimum: name,
description, featured image, technologies, link to live version, link to source.
-- update the main page so that the projects section is created dynamically using the information stored
in that JavaScript object.
All of the HTML in that section is created when the page loads.
implement the popup window.
-- both mobile and desktop screen sizes

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -656,7 +659,8 @@ ul {
   /* brought from ofm (start) */
   .header {
     padding-top: 10px;
-    position: fixed !important;
+
+    /* position: fixed !important; */
```
Comment (Deepakdanger) [inline css/styles.css]: - [ ] kindly delete the commented code.

File: css/styles.css [hunk 2]
```diff
@@ -746,22 +750,29 @@ ul {
...
     row-gap: 140px;
     background-color: var(--gridBackground);
     grid-template-columns: auto;
-    padding-top: 0;
+    padding-top: 140px;
     padding-bottom: 140px;
     padding-left: 24px;
     padding-right: 24px;
   }
+  */
```
Comment (Deepakdanger) [inline css/styles.css]: - [ ] kindly delete the commented code.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @ivanmvh , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @ivanmvh, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**